bundles/org.openhab.core.io.transport.modbus/pom.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.core.bundles</groupId>
+    <artifactId>org.openhab.core.reactor.bundles</artifactId>
+    <version>3.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.core.io.transport.modbus</artifactId>
+
+  <name>openHAB Core :: Bundles :: Modbus Transport</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.openhab.core.bundles</groupId>
+      <artifactId>org.openhab.core.config.core</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.openhab.core.bundles</groupId>
+      <artifactId>org.openhab.core.test</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-pool2</artifactId>
+      <version>2.8.1</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>net.wimpi</groupId>
+      <artifactId>jamod</artifactId>
+      <version>1.2.4.OH</version>
+      <scope>compile</scope>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>biz.aQute.bnd</groupId>
+        <artifactId>bnd-maven-plugin</artifactId>
+        <version>${bnd.version}</version>
+        <configuration>
+          <bnd><![CDATA[Bundle-SymbolicName: ${project.artifactId}
+Automatic-Module-Name: ${def;bsn}
+Import-Package: \\
+  org.eclipse.jdt.annotation.*;resolution:=optional,\\
+  org.openhab.*;version=!,\\
+  !net.sf.cglib.proxy,\\
+  gnu.io;version="[3.12,6)",\\
+  *
+-sources: false
+-contract: *]]></bnd>
```

Comment:
Would it be possible to move this to a `bnd.bnd` file? That's where we manage these customizations for the other bundles in openhab-core.

Reply:
If you can help me figuring out, what content should be put in that bnd file...? Exactly what is here in the `<bnd>` element?

Reply:
Seems to work - done.

Reply:
Does it also work with one trailing `\`? That's what is used in all the other [bnd.bnd](https://github.com/openhab/openhab-core/blob/master/bundles/org.openhab.core.model.item/bnd.bnd) files. They also all use only one space for indentation.

Reply:
sure, done.